File: user/src/main/java/com/undancer/app/UserController.java
package com.undancer.app;

import com.google.common.collect.Maps;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.Map;

@RestController
@RequestMapping(path = "/user")
public class UserController {

    @GetMapping
    public Object get() {
        Map<String, Object> resultObject = Maps.newHashMap();
        resultObject.put("username", "undancer");
        return resultObject;
    }

}

File: services/user/src/main/kotlin/com/undancer/cloud/user/UserController.kt
package com.undancer.cloud.user

import com.google.common.collect.Maps
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping(path = ["/user"])
class UserController {

    @GetMapping
    fun get(): Any {
        val resultObject = Maps.newHashMap<String, Any>()
        resultObject["username"] = "undancer"
        return resultObject
    }

}
